pyproject.toml

```diff
@@ -24,7 +24,7 @@
 requires = [
     "GitPython==3.1.44",
     "gitdb==4.0.12",
-    "hatchling==1.27.0",
+    "hatchling==1.25.0",
```

Comment:
Why do you downgrade hatchling? Does the new version have a bug?

Reply:
Just to synchronize it across the board. We did not need to have 1.27 :)